### PR TITLE
LPD-36951 Use quantity selector

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-36951

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js [hunk 1]
```diff
@@ -19,15 +19,29 @@ import {
...
 
-	if (!value || value < min) {
-		errors.push('min');
+	if (allowEmptyValue) {
+		if (value && value < min) {
+			errors.push('min');
+		}
+	}
+	else {
+		if (!value || value < min) {
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js:42]: Nesting not needed.

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js [hunk 2]
```diff
@@ -19,15 +19,29 @@ import {
 } from '../../utilities/quantities';
 import RulesPopover from './RulesPopover';
 
-const getErrors = (value, min, max, step, precision = 0) => {
+const getErrors = (
+	value,
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/InputQuantitySelector.js:23]: [NB]: arguments have become many, being local we could consider converting from separate function arguments to a single object.

File: modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/ListQuantitySelector.js [hunk 3]
```diff
@@ -8,21 +8,33 @@ import React, {forwardRef} from 'react';
...
 				disabled={disabled}
 				name={name}
 				onChange={({target}) => {
 					onUpdate({errors: [], value: Number(target.value)});
 				}}
-				options={allowedQuantities.map((value) => ({
-					label: String(value),
-					value,
-				}))}
+				options={(allowEmptyValue ? [''] : [])
```
Comment (gianmarcobrunialti) [inline modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/quantity_selector/ListQuantitySelector.js:32]: NB: 🥹

**Review by gianmarcobrunialti — CHANGES_REQUESTED** [review]
Just 1 minor thing.

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:stop

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:forward

**Pull request closed without merging**